### DISCUSSION TITLE
Episode 198: Talks ∧ ¬ (Pigeons ∧ Elevators)

### DISCUSSION THREAD

**Discussion by codereport** [codereport/adsp2#97] [Episode]
Episode Link: https://adspthepodcast.com/2024/09/06/Episode-198.html

Feel free to write comments or ask questions 🙂

**Comment by amazingmo** [comment]
Hi Conor,
You don't know me so I don't expect anything but can I ask you for a personal favour please?
When you want to elide a section of conversation from the recording, can you please just cut it rather than playing forward at 10x?
My hearing isn't great at normal speech frequencies so I have the volume bumped up. The 10x talk cuts through like a knife though. It's really unpleasant. There's no content in it so can you just not have it in the recording?
If you're worried about the listeners being confused you could have some short, subtle audible cue to signal that you've cut out off-topic content.
Thanks for reading this.

**Reply by codereport** [reply]
The good news is there won't be much 10x-ing for a while. That being said I know folks have said it is funny, but maybe I will de-amplify the sped up bit so it is much quieter. If I remember to do this and it is still harsh, let me know and I can consider replacing it with something different.

**Reply by amazingmo** [reply]
Yes, it was funny the first time, but the humor diminishes with each successive use, and _especially_ for long blocks.
If you really want to keep the squeak, maybe cut it down to 1 second?  You'll get a laugh (maybe), but you won't be wasting everyone's time with long tracts of unintelligible auditory assault.
Thanks!

**Comment by The-Amoghavarsha** [comment]
Hey Conor, first you baited me by using **∧ ¬** symbols. Before clicking them, I presumed that they were going to be on functional programming (that's capital lambda there!!) or array programming, or a juxtaposition of both. Fine, I can live with that. But, why did you fast-forward Bryce's Pigeon story? 😭😭😭

**Reply by codereport** [reply]
Haha, trust me the pigeon story wasn't that excited. And I didn't intend to tricka anyone :see_no_evil: Those are valid BQN symbols :slightly_smiling_face:
